mesheryctl/pkg/utils/helpers.go

```diff
@@ -1220,15 +1220,6 @@ func CheckFileExists(name string) (bool, error) {
 	return false, errors.Wrap(err, fmt.Sprintf("Failed to read/fetch the file %s", name))
 }
 
-func Contains(key string, col []string) int {
```

Comment:
I assume this functionality has been relocated to meshkit in the `registry.go` file. Please verify if there's an existing equivalent function in Meshkit. If so, use that instead of migrating this code.